vscode-trace-extension/src/trace-explorer/trace-tree.ts

```diff
@@ -164,7 +208,21 @@ export const fileHandler = (analysisTree: AnalysisProvider) => async (context: v
             }
         }
 
-    })();
+        if (token.isCancellationRequested) {
+            if (experiment) {
+                experimentManager.deleteExperiment(experiment.UUID);
+            }
+            // Rollback traces
+            progress.report({ message: 'Rolling back trace(s)',  increment: 20});
```

Comment:
Could this duplicating message string be a reused constant instead? So that we consistently maintain it if need be; we could otherwise forget one. This may apply also to other non-trivial messages in this change.

In fact, maybe rather extract this whole duplicated rollbacking logic in a reused function?